shakenfist/net.py

```diff
@@ -40,6 +40,11 @@ def from_db(uuid):
                    namespace=dbnet['namespace'])
 
 
+def is_network_node():
```

Comment:
I wonder if this should be in shakenfist.util?

Reply:
Sure. Will do.

Reply:
I didn't replace all the other uses to have a minimal touch for patching CT.
Can use it more widely in a later PR.